pkg/universe.dagger.io/x/benjamin.reigner@epitech.eu/gcp/gcloud.cue

```diff
@@ -0,0 +1,52 @@
+package gcp
+
+import (
+	"universe.dagger.io/alpine"
+	"universe.dagger.io/docker"
+)
+
+#GCloud: {
```

Comment:
See:
- https://docs.dagger.io/1226/coding-style#all-public-definitions-and-fields-must-be-documented
- https://docs.dagger.io/1226/coding-style#avoid-interpolation
- https://docs.dagger.io/1226/coding-style#avoid-raw-strings